libraries/extractor/src/main/java/androidx/media3/extractor/mp4/AtomParsers.java

```diff
@@ -1122,6 +1123,133 @@ private static void parseTextSampleEntry(
             .build();
   }
 
+  // Helper class for parsing syntax elements from AV1 bitstream.
+  // Bitstream specification: https://aomediacodec.github.io/av1-spec/av1-spec.pdf
+  private static final class Av1BitstreamParser {
+    private static final String TAG = "Av1BitstreamParser";
+
+    private ParsableByteArray data;
```

Comment:
This can be a final field as its set only once in the constructor.

Reply:
done.